### PR TITLE
Migrate to GitHub Actions [master]

### PR DESCRIPTION
**Pull request by tylerjw**
Creating the PR here on the repo to get the pre-commit checks to run here.

### CODE REVIEW

**Review by rhaschke — APPROVED** [review]
Do you want to stick with Travis here? Why?

**Comment by tylerjw** [comment]
> Do you want to stick with Travis here? Why?

No, I'll add that now to this.  I was going to do it in a separate PR but now I feel silly for not just doing it here.

File: .github/workflows/industrial_ci_action.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - ROS_DISTRO: melodic
+            ROS_REPO: main
+          - ROS_DISTRO: melodic
+            ROS_REPO: testing
+            CATKIN_LINT: true
```
Comment (rhaschke) [inline .github/workflows/industrial_ci_action.yaml:23]: Let's add Noetic to the matrix as well:
```suggestion
          - ROS_DISTRO: noetic
            ROS_REPO: testing
            CATKIN_LINT: true
          - ROS_DISTRO: melodic
            ROS_REPO: main
```

**Review by rhaschke — COMMENTED** [review]
Thanks. Please add a Noetic build as suggested.

**Pull request merged**